traces/random.txt

```diff
@@ -0,0 +1,12 @@
+SUB R1 R2 R3
+ADD R4 R5 R6
+ADD R7 R8 R9
+ADD R10 R11 R12
+ADD R13 R14 R15
+XOR R30 R31 R31
+ADD R6 R4 R5
+ADDI R4 R2 0x2151
+SUBI R4 R2 0x2151
```

Comment:
Reece made note of the issue with hex here as well, but to confirm, I updated these 2 lines to have a decimal number in place of the hex format, and it worked as expected.